libs/model/src/models/community.ts

```diff
@@ -67,20 +67,20 @@ export default (
         allowNull: false,
         defaultValue: [],
       },
-      default_symbol: { type: dataTypes.STRING, allowNull: false },
+      default_symbol: { type: dataTypes.STRING, allowNull: true },
       network: { type: dataTypes.STRING, allowNull: false },
       base: { type: dataTypes.STRING, allowNull: false, defaultValue: '' },
       ss58_prefix: { type: dataTypes.INTEGER, allowNull: true },
       icon_url: { type: dataTypes.STRING },
       active: { type: dataTypes.BOOLEAN },
       stages_enabled: {
         type: dataTypes.BOOLEAN,
-        allowNull: false,
+        allowNull: true,
         defaultValue: true,
       },
       custom_stages: {
         type: dataTypes.ARRAY(dataTypes.TEXT),
-        allowNull: false,
+        allowNull: true,
```

Comment:
Default value here is actually incorrect per schema, it should be whatever `'{}'::text[]` resolves to, which seems to NOT be `[]` but `'{}'`.

Reply:
The model won't compile with a default value that is not an array... guess the migration is bad?

Reply:
Yeah something weird is happening here. Should we change anything or leave it as is / as a weird exception? @Rotorsoft

Reply:
I would leave it as is... seems like this is not causing any problems ... @timolegros ?

Reply:
@jnaviask @Rotorsoft I believe '{}' is shorthand for ARRAY[]. Both initialize an empty Postgresql array => {}.